contracts/truefi2/TrueCreditAgency.sol

```diff
@@ -50,6 +50,8 @@ contract TrueCreditAgency is UpgradeableClaimable {
 
     ITrueFiCreditOracle public creditOracle;
 
+    uint256 public usedBuckets;
```

Comment:
Let's add a descriptive comment explaining this is a bitmap and why our uint8 credit scores make it okay to store the bitmap in a uint256. Otherwise readers could get confused and think this is a count.

Probably also include Bitmap in the name, e.g., `usedBucketsBitmap`. Though if you prefer this naming then I wouldn't mind.

Reply:
If you have profiling results, this might also be a good place to document how much gas a bitmap empirically saves compared to a direct mapping.